stock_picking_move_description/security/stock_picking_move_description_security.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="group_picking_report" model="res.groups">
+            <field name="name">Picking report: hide product column</field>
```

Comment:
Change ID and name, according to actual usage, and better name picking report with official Odoo description (Delivery Slip):
```suggestion
        <record id="group_picking_report_show_prod" model="res.groups">
            <field name="name">Delivery Slip: show product column</field>
```